tests/tests/swfs/avm2/vector3d/Test.as

```diff
@@ -0,0 +1,322 @@
+﻿package {
+    public class Test {
+    }
+}
+
+import flash.geom.Vector3D;
+
+
+/*
```

Comment:
Did you mean to include this?

Reply:
This is all very much a draft, that's there temporarily, as a reminder for the list of methods to test and their parameter lists.

Reply:
Sorry, I didn't see that this was a draft :smile:

Reply:
I pushed it just to avoid any potential duplicate effort.